website/docs/cdktf/release/upgrade-guide-v0-15.mdx

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Upgrading to CDKTF Version 0.15
+description: >-
+  Minimum version of Terraform increased to 1.2, classes reanamed for C#, Java, and Go
+---
+
+# Upgrading to CDK for Terraform Version 0.15
+
+0.15 is primarily focused around CLI improvements targeted at a better Terraform Cloud integration.
+Because of this, there are only smaller breaking changes in this release. The biggest one is a refactoring of several Constructs in our core library.
```

Comment:
I'd make it a bit more weak, as this is a minor breaking change but the sentence sounded a bit extreme 😁
Maybe something like this?
```suggestion
Because of this, there are only smaller breaking changes in this release. The most notable one is a renaming of some interfaces in our core library.
```